i18n/it_IT.json

```diff
@@ -79,24 +79,24 @@
     "Coupon Code": "Codice Coupon",
     "Coupon was applied!": "È stato utilizzato il codice coupon.",
     "Coupon was removed!": "Hai cancellato il codice coupon.",
-    "Create Password": null,
+    "Create Password": "Crea Password",
     "Create an account": "Crea un Account",
-    "Create free account and keep track of your orders": null,
-    "Create new account": null,
+    "Create free account and keep track of your orders": "Crea un account gratuito e tieni traccia dei tuoi ordini",
+    "Create new account": "Crea nuovo account",
     "Created at:": "Creato il:",
-    "Created at: ": null,
+    "Created at: ": "Creato il:",
     "Current Password": "Password attuale",
     "Current product price": "Prezzo attuale del prodotto",
     "Current store:": "Negozio attuale:",
     "Customer reviews": "Recensioni dei clienti",
     "Dashboard": "Pannello di controllo",
-    "Date": null,
+    "Date": "Data",
     "Default %s address": "Indirizzo predefinito",
     "Delete": "Elimina",
     "Description:": "Descrizione:",
-    "Description: ": null,
+    "Description: ": "Descrizione:",
     "Don't have an account?": "Non hai un account?",
-    "Don`t have an account?": null,
+    "Don`t have an account?": "Non hai un account?",,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character (',' (code 44)): was expecting double-quote to start field name](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=5403666)

Reply:
Sorry for that, pushed a fix.